src/hardware/pcspeaker.cpp

```diff
@@ -437,21 +424,28 @@ static float impulse(const double t)
 	const auto fc = fs / (2 + static_cast<double>(SPKR_CUTOFF_MARGIN));
 	const auto q = static_cast<double>(SPKR_FILTER_QUALITY);
 	if ((0 < t) && (t * fs < q)) {
-		const auto window = 1.0 +
-		                    cos(2 * fs * M_PI * (q / (2 * fs) - t) / q);
-		const auto level = window *
-		                   (sinc(2 * fc * M_PI * (t - q / (2 * fs)))) / 2.0;
-		return static_cast<float>(level);
+		const auto window = 1.0 + cos(2 * fs * M_PI * (q / (2 * fs) - t) / q);
+		const auto amplitude = window * (sinc(2 * fc * M_PI * (t - q / (2 * fs)))) / 2.0;
+		return static_cast<float>(amplitude);
 	} else
 		return 0.0f;
 }
 
-static void add_impulse(const DelayEntry &entry)
+static void AddImpulse(float index, const int16_t amplitude)
```

Comment:
Ok, looks this another questions of mine has been answered: we should use UpperCamelCase for method/function names, apparently.

Reply:
Yeah; the original code used CamelCase, then ripa's patch added snake_case. 
When I refactor it into a class, they'll all be member functions CamelCase.